pkg/users/accounts-list.js

```diff
@@ -109,7 +109,10 @@ const getGroupRow = (group, accounts, isUserCreatedGroup) => {
                 <Flex alignItems={{ default: 'alignItemsCenter' }}>
                     <div className={"dot " + groupColorClass} />
                     <FlexItem>
-                        <a href={"#/group/" + group.name}>{group.name}</a>
+                        <a onClick={ev => { cockpit.jump("/users#/group/" + group.name); ev.preventDefault() }}
```

Comment:
I debugged a similar issue on the Services page in #18443. Similar scenario, there's a "details" view controlled by cockpit.location.path with usePageLocation. I fixed that [like this](https://github.com/cockpit-project/cockpit/pull/18445/commits/3498eeca6848fd6a927f8abed76e971ee6953b58). I believe the Users page has a similar problem: 
 - The UserActions menu calls `onClick={ev => { ev.preventDefault(); cockpit.location.go(account.name) }}>` for "Edit user"
 - This triggers usePageLocation(), and thus the two useEffect handlers of AccountsList and GroupsList
 - both handlers do `cockpit.location.go([], newOptions)`, i.e. they reset the path back from `#/accountname` to `#/`

IMHO, fixing these two to use `cockpit.location.path` instead of `[]` is a proper fix, while this is both rather hackish (hardcoding `/users` and thus changing frames and messing up history), and I feel this more or less works just by accident, as it is more expensive and thus takes longer, and thus the useEffect() handlers might just ran a bit faster.

I assume you get a lot of test failures without this commit? Do you have a URL? Could you try the solution above?

Reply:
I don't have a URL for the failures, but yes without the last commit it failed on moving to details page.

Tried your solution instead!

Reply:
Hmm it seems it did not help the fix from the services page,. https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230306-093144-8600b065-fedora-37-firefox/log.html#338